doc/api/util.md

```diff
@@ -126,6 +126,10 @@ util.format(1, 2, 3); // '1 2 3'
 
 ## util.inherits(constructor, superConstructor)
 
+_Note: usage of `util.inherits()` is discouraged. Please use ES6 `class` and
+`extends` to get language level inheritance support, including an unbroken
+inheritance chain._
```

Comment:
I probably need your help on this, since I think (but am not sure) the usage of .setPrototypeis wrong, not getting prototype correctly. What it actually does instead I have no idea.